<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWindowSize</class>
 <widget class="QDialog" name="DialogWindowSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial Window Size</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter desired size in rows and columns of characters:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="editRows"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editColumns"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogWindowSize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
